open-hackathon/src/create_test_data.py

```diff
@@ -53,4 +69,5 @@
             provider = VirtualEnvironmentProvider.AzureVM
         db_adapter.add_object_kwargs(Template, hackathon=h, name=name, url=template_url, provider=provider)
```

Comment:
please reformat all your changed files. The indent of the above two lines is not right.